sentences/de/_common.yaml

```diff
@@ -69,9 +69,9 @@ lists:
         out: "lock"
   lock_states:
     values:
-      - in: "(([ab|zu]ge|ver)schlossen|verriegelt)"
+      - in: "(([ab|zu]ge|ver)schlossen|verriegelt|((ab|zu)ge|ver)sperrt)"
         out: "locked"
-      - in: "(nicht ([ab|zu]ge|ver)schlossen|entriegelt)"
+      - in: "(nicht (([ab|zu]ge|ver)schlossen|verriegelt|((ab|zu)ge|ver)sperrt)|entriegelt|entsperrt)"
```

Comment:
Should now additionally accept phrases like:
nicht verriegelt
nicht verperrt
nicht abgesperrt
nicht zugesperrt
entsperrt